### PR TITLE
Reorg Profile's Cypress tests

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Some of the test file names were looooong. This PR groups related tests together in folders which also allows us to shorten their names.

## Testing done
As long as these tests still pass we're good. 🤞

## Screenshots
BEFORE
image

AFTER
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
-import { PROFILE_PATHS } from '../../constants';
+import { PROFILE_PATHS } from '@@profile/constants';
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js:2]: I'm fairly sure that `@@profile` is better than `../../../`

File: src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-import mockUserNotInEVSS from '../fixtures/users/user-non-vet.json';
-import mockUserInEVSS from '../fixtures/users/user-36.json';
+import mockUserNotInEVSS from '@@profile/tests/fixtures/users/user-non-vet.json';
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js:4]: I'm less sure if `@@profile/tests` is better than `../../`. Thoughts?

File: src/applications/personalization/profile/tests/fixtures/users/user-not-in-mpi.json [hunk 3]
```diff
@@ -3,18 +3,7 @@
     "id": "",
     "type": "users_scaffolds",
     "attributes": {
-      "services": [
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/fixtures/users/user-not-in-mpi.json:6]: Not at all related to the main point of this PR, but I'm going to start cleaning up mock users when I see them with lots of unnecessary properties.

File: src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js (hunk 2 above)
Comment (sanlouise) [reply src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js:4]: Definitely better IMO to use the alias.

**Review by sanlouise — APPROVED** [review]
Looks good to me!

**Pull request merged**